src/Symfony/Component/AssetMapper/Tests/ImportMap/ImportMapManagerTest.php

```diff
@@ -373,6 +373,65 @@ public function testUpdate()
         $this->assertSame('contents of cowsay.js', $actualContents);
     }
 
+    public function testUpdateWithSpecificPackages()
+    {
+        $rootDir = __DIR__.'/../fixtures/importmaps_for_writing';
+        $manager = $this->createImportMapManager(['assets' => ''], $rootDir);
+
+        $map = [
+            'lodash' => [
+                'url' => 'https://ga.jspm.io/npm:lodash@1.2.3/lodash.js',
+            ],
+            'cowsay' => [
+                'url' => 'https://ga.jspm.io/npm:cowsay@4.5.6/cowsay.umd.js',
+                'downloaded_to' => 'vendor/moo.js',
+            ],
+            'bootstrap' => [
+                'url' => 'https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/js/bootstrap.esm.js',
+                'preload' => true,
+            ],
+            'app' => [
+                'path' => 'app.js',
+            ],
+        ];
+        $mapString = var_export($map, true);
+        file_put_contents($rootDir.'/importmap.php', "<?php\n\nreturn {$mapString};\n");
+        $this->filesystem->mkdir($rootDir.'/assets/vendor');
+        file_put_contents($rootDir.'/assets/vendor/moo.js', 'moo.js contents');
```

Comment:
```suggestion
        file_put_contents($rootDir.'/assets/vendor/moo.js', 'moo.js original contents');
```

or, better `cowsay.js original contents` if you change the filename